src/ManojN/Q3checkNumberInArray.java

```diff
@@ -0,0 +1,26 @@
+/*
+3) Check if array contains a number in Java?
+Hint: Your method should accept 2 parameters. Input array and number.
+i/p :- arr[] = {2,5,9,7,4,6}
+number :- 8
+o/p :- number is not in arr.
+ */
+package a2;
+public class Q3checkNumberInArray {
+	static void m1(int arr1[], int num){
+		//int arr[] = {2,5,9,7,4,6};
+		for(int i=0; i<arr1.length; i++){
+			if (num==arr1[i]){
+			System.out.println("num is same");
+			
+			}	else if (num!=arr1[i]){
```

Comment:
Wrong logic manoj here..
check your code for any existing the num value it print that num as missing...which is wrong because of else condition..

And for num 2 it print
num is same
missing no from array : 2